fixed external.pro file (missing linebreak and file-ending)
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 meshlab (1.3.3) precise; urgency=low
 .
   * new snapshot release
Author: Stefan Schlager <zarquon42@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- meshlab-1.3.3.orig/meshlab/src/external/external.pro
+++ meshlab-1.3.3/meshlab/src/external/external.pro
@@ -7,6 +7,6 @@ SUBDIRS	= 	bzip2-1.0.5/bzip2-1.0.5.pro \
 			levmar-2.3/levmar-2.3.pro \
 			structuresynth/structuresynth.pro \
 			OpenCTM-1.0.3/openctm.pro \
-			jhead-2.95/jhead-2.95.pro
-			lib3ds-1.3.0/lib3ds
+			jhead-2.95/jhead-2.95.pro \
+			lib3ds-1.3.0/lib3ds/lib3ds.pro
 #			openkinect/openkinect.pro
